From: Boyuan Yang <073plan@gmail.com>
Date: Sat, 17 Sep 2016 16:30:23 +0800
Subject: setup.py: fix version number
Forwarded: not-needed

Upstream forgot to update version number from 2.8.2 to 2.9.0.
Making patch to fix it.

I am not cherry-picking it because the upstream fixed it
in a large commit (6ef14e56).

Signed-off-by: Boyuan Yang <073plan@gmail.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e6a7ff7..ba61ad3 100644
--- a/setup.py
+++ b/setup.py
@@ -7,7 +7,7 @@ with codecs.open('README.rst', encoding='utf-8') as f:
 
 setup(
     name="shadowsocks",
-    version="2.8.2",
+    version="2.9.0",
     license='http://www.apache.org/licenses/LICENSE-2.0',
     description="A fast tunnel proxy that help you get through firewalls",
     author='clowwindy',
